kinto/views/collection.py

```diff
@@ -0,0 +1,10 @@
+from cliquet.resource import crud, BaseResource
+
+
+@crud(path="/collections/{collection_id}/records/{id}",
+      collection_path="/collections/{collection_id}/records")
+class Collection(BaseResource):
+
+    @property
+    def name(self):
+        return self.request.matchdict['collection_id']
```

Comment:
Ah yes, that's because of https://github.com/mozilla-services/cliquet/issues/30

collections management is definitely not a requirement :)